src/code/z_viscvg.c

```diff
@@ -75,20 +109,29 @@ void func_800ACE98(struct_801664F0* this, Gfx** gfxp) {
     }
 
     switch (this->type) {
-        case 1:
-            gSPDisplayList(gfx++, D_8012AC40);
+        case FB_FILTER_CVG_RGB:
+            gSPDisplayList(gfx++, sCoverageRGBDL);
             break;
-        case 2:
-            gDPSetColor(gfx++, G_SETPRIMCOLOR, this->color.rgba);
-            gSPDisplayList(gfx++, D_8012AC58);
+
+        case FB_FILTER_CVG_RGB_PRIMFOG:
+            // Set primitive color for uniform fogging in custom RenderMode
+            gDPSetColor(gfx++, G_SETPRIMCOLOR, this->primColor.rgba);
+            gSPDisplayList(gfx++, sCoverageRGBPrimFogDL);
```

Comment:
I'm still not a fan of using "fog" at a low level when fog isn't involved

`PRIMOVERLAY` could be an idea maybe, or `primcoat`, `primcover` (thanks synonyms dictionaries)

Reply:
In lighting this would be called a "gel", I think `// Set primitive color for uniform color filter in custom RenderMode` is probably fine

Reply:
Sure

Reply:
(the comment was also on the enum name and dlist name though)